frontend/workflows-lib/lib/utils/validationUtils.ts

```diff
@@ -0,0 +1,102 @@
+import { ScanRange } from "../types";
+
+export const isStrictPositiveInteger = (value: string): boolean =>
+  /^\d+$/.test(value) && Number(value) >= 0;
+
+export const parseExcludedScans = (input: string): number[] => {
+  const cleaned = input.replace(/[[\]]/g, "");
+
+  const parts = cleaned
+    .split(",")
```

Comment:
I think that this requires input in the form "1, 2, 3" etc . Do we want to be more permissive and accept other separators (eg any whitespace)?

Reply:
I've added whitespace separation.